### PR TITLE
Move release into pipeline

### PR DESCRIPTION
**Pull request by damccorm**
Sample run:
https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=11588943&view=logs&j=614b717c-6df9-52e7-0d76-708a38eacb97

This generates PRs like:
- https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps/pullrequest/537720
- https://dev.azure.com/mseng/AzureDevOps/_git/AzureDevOps.ConfigChange/pullrequest/537721

### CODE REVIEW

File: .vsts.release.yml [hunk 1]
```diff
@@ -295,3 +318,29 @@ stages:
...
+        set -x
+        git checkout releases/$(version)
+        cd release
+        npm install
+        ls
+        node createAdoPrs.js $(version)
+      displayName: Create PRs in AzureDevOps and AzureDevOps.ConfigChange
+      condition: ne(variables['version'], '')
+      env:
+        USER: 'damccorm'
```
Comment (jtpetty) [inline .vsts.release.yml]: can we use a service account or make this a variable along side the AdoPAT?

File: .vsts.release.yml [hunk 2]
```diff
@@ -295,3 +318,29 @@ stages:
...
+
+- stage: CreatePRs
+  dependsOn: Release
+  jobs:
+  ################################################################################
+  - job: create_ado_prs
+  ################################################################################
+    displayName: Create PRs in AzureDevOps
+    pool:
+      vmImage: ubuntu-16.04
```
Comment (jtpetty) [inline .vsts.release.yml]: doesn't matter a whole lot, but maybe we should run these on 18.04 instead of 16.04?

File: .vsts.release.yml (hunk 1 above)
Comment (damccorm) [reply .vsts.release.yml]: I want to move it to a service account, but wanted to get it up and running first before dealing with that. Agreed on making it a variable though - updated

File: .vsts.release.yml (hunk 2 above)
Comment (damccorm) [reply .vsts.release.yml]: Done

**Pull request merged**